main.go

```diff
@@ -110,9 +110,9 @@ func Main() (retcode int) {
 	}
 	defer func() {
 		if hookAfter != nil {
-			retcode = hookAfter()
+			retcode += hookAfter()
```

Comment:
could use retcode |= maybe instead of += but we don't expect more than 254 errors

Reply:
You know it could be an issue.

I don't think you need the feature to return a code that count things.

You have logs